src/sbt-test/sbt-daffodil/saved-parsers-04/build.sbt

```diff
@@ -24,7 +24,7 @@ organization := "com.example"
 enablePlugins(DaffodilPlugin)
 
 daffodilPackageBinInfos := Seq(
-  DaffodilBinInfo("/does-not-exist.dfdl.xsd")
+  DaffodilBinInfo("/does-not-exist.dfdl.xsd"),
```

Comment:
Adding the trailing comma is a departure from main daffodil's scalafmt, is that on purpose?

Reply:
Not intentional, these projects used to use the same config but then we decided on changes that forgot to make to this one. I'll update the config so they match.